src/enclave-agent/Cargo.toml

```diff
@@ -20,7 +20,7 @@ ctrlc = { version = "3.0", features = ["termination"] }
 tokio = { version = "1.14.0", features = ["full"] }
 async-trait = "0.1.42"
 
-image-rs = { git = "https://github.com/confidential-containers/image-rs", tag = "v0.2.0", features = ["occlum_feature"], default-features = false }
+image-rs = { git = "https://github.com/confidential-containers/image-rs", features = ["occlum_feature"], default-features = false }
```

Comment:
we want to use a known `rev` here

Reply:
`rev` has been added.

Reply:
thanks, let's merge #80 first and then rebase this on top of it